components/brave_wallet/common/polkadot_utils.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_COMMON_POLKADOT_UTILS_H_
+#define BRAVE_COMPONENTS_BRAVE_WALLET_COMMON_POLKADOT_UTILS_H_
+
+namespace brave_wallet {
+
+// See definition for "path": ["sp_core", "crypto", "AccountId32"]
+// https://raw.githubusercontent.com/polkadot-js/api/refs/heads/master/packages/types-support/src/metadata/v16/substrate-types.json
+inline constexpr const size_t kPolkadotSubstrateAccountIdSize = 32;
```

Comment:
Will it be used outside /browser/? "common" directory is used to share between renderer and browser. 
I know there are a lot of unrelated files for now in the brave_wallet/comon/ directory but anyway please
consider moving to /browser/ instead

Reply:
It probably won't be used outside browser, so I moved it to `browser/polkadot/` and it seems like `npm run gn_check` is happy with that, which was my original motivation anyway.